src/api/app.py

```diff
@@ -186,7 +186,10 @@ async def logging_middleware(request: Request, call_next) -> Response:
     return response
 
 
-langfuse_handler = CallbackHandler()
+os.environ["LANGFUSE_TRACING_ENVIRONMENT"] = os.getenv("STAGE", "production")
```

Comment:
am still a bit concerned by the extra lines / changes here - but if you think it's okay, happy to merge